netbox_lifecycle/templates/netbox_lifecycle/hardwarelifecycle.html

```diff
@@ -23,7 +24,7 @@ <h5 class="card-header">{{ object.assigned_object_type.name|capfirst}}</h5>
             </tr>
             <tr>
               <th>Description</th>
-              <td>{{ object.description }}</td>
+              <td>{{ object.assigned_object.description }}</td>
```

Comment:
This should be left as is, description was added to all models, including hardware lifecycle.

Reply:
All good, was interpreting the description was related to the hardware device as it was part of the hw device card vs elsewhere.  Reverted and pushed.